style.css

```diff
@@ -1,57 +1,72 @@
-body, html {
-    height: 100%;
-    margin: 0;
-    overflow: hidden;
-    background-color: black;
-    color: white;
-    position: absolute;
-	left: 0;
-	top: 0;
-    font-family: 'Roboto Mono', monospace;
+body,
+html {
+  height: 100vh;
+  margin: 0;
+	width: 100vw;
+  overflow: hidden;
+  background-color: black;
+  color: white;
+  position: absolute;
+  left: 0;
+  top: 0;
+  font-family: "Roboto Mono", monospace;
 }
 
 #matrixContainer {
-	display: flex;
+  display: flex;
 }
 
 .stream {
-    font-size: 24px;
-	width: 24px;
-	text-align: center;
-	display: flex;
-	flex-direction: column;
-	color: white;
+	-webkit-user-select: none;
+	-moz-user-select: none;
+	-ms-user-select: none;
+	user-select: none;
+  font-size: 24px;
+  width: 24px;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  color: white;
+  min-width: 1ch;
```

Comment:
Note to Avi: 1ch is the width of a zero for a given font family, in this case, because it's a mono font, this should handle all char widths